tensorrt_llm/_torch/pyexecutor/config.py

```diff
@@ -50,6 +50,8 @@ class PyTorchConfig:
     attention_dp_time_out_iters: int = 50
     attention_dp_batching_wait_iters: int = 10
 
+    batch_wait_timeout: float = 0
```

Comment:
It seems that the unit of batch_wait_timeout is second, can we use micro-second instead? 

```python
batch_wait_timeout_ms: int = 0
```